docs/elasticsearch/index.md

```diff
@@ -52,25 +52,25 @@ When you add the document to an index, Elasticsearch adds some metadata, such as
+Older versions of Elasticsearch used arbitrary document *types*, but indices created in current versions of Elasticsearch should use a single type named `_doc`. You must store different document types in different indices.
 {: .note }
 
 
 ## Primary and replica shards
 
-Elasticsearch splits indices into *shards* for even distribution across nodes in a cluster. For example, a 400 GB index might be too large for any single node in your cluster to handle, but split into ten shards, each one 40 GB, Elasticsearch can distribute the shards across ten nodes and work with each shard individually.
+Elasticsearch splits indices into *shards* for even distribution across nodes in a cluster. For example, a 400 GB index might be too large for any single node in your cluster to handle. But when split into ten shards, 40 GB each, Elasticsearch can distribute the shards across ten nodes and work with each shard individually.
 
-By default, Elasticsearch creates a *replica* shard for each *primary* shard. If you split your index into ten shards, for example, Elasticsearch also creates ten replica shards. These replica shards act as backups in the event of a node failure---Elasticsearch distributes replica shards to different nodes than their corresponding primary shards---but they also improve the speed and rate at which the cluster can process search requests. You might specify more than one replica per index for a search-heavy workload.
+By default, Elasticsearch creates a *replica* shard for every *primary* shard. If you split your index into ten shards, for example, Elasticsearch also creates ten replica shards. These replica shards act as backups in the event of a node failure. Elasticsearch distributes these replica shards to other nodes, different from the nodes of the primary shards. This improves the speed and rate at which the cluster can process search requests. You can specify more than one replica shard per index for a search-heavy workload.
 
-Despite being a piece of an Elasticsearch index, each shard is actually a full Lucene index---confusing, we know. This detail is important, though, because each instance of Lucene is a running process that consumes CPU and memory. More shards is not necessarily better. Splitting a 400 GB index into 1,000 shards, for example, would place needless strain on your cluster. A good rule of thumb is to keep shard size between 10--50 GB.
+Despite being a piece of an Elasticsearch index, each shard is actually a full Lucene index. This is important because each instance of Lucene is a running process that consumes CPU and memory. More shards are not necessarily better. Splitting a 400 GB index into 1,000 shards, for example, would place needless strain on your cluster. A good rule of thumb is to keep shard size between 10--50 GB.
```

Comment:
So I spent a non-trivial amount of time trying to find a grammar recommendation for "more shards are" vs. "more shards is." If you're able to find something authoritative, please let me know. Overall, I think the subject here is still "more" rather than "shards," because the key concept is the idea of more rather than some number of shards. Recommend leaving as-is unless you can find a grammar reference that proves me wrong, which I will happily accept.

We can also just reword the sentence to something like "Increasing shard count is often a bad idea." or "Use caution when increasing shard count."